src/app.ts

```diff
@@ -0,0 +1,477 @@
+import config from "config";
```

Comment:
next time do a git mv myile.js my file.ts

Reply:
ah, OK - I guess that allows for tracking to follow?